stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.2-a
+version: 1.2.0-a
```

Comment:
The version scheme this particular chart has observed, to date, is that we are in lock step with the semver version number of Traefik itself, and we append `-a`, `-b`, etc. to denote versions of the _chart_ for that given Traefik release. So I'd rather see this change to `1.1.2-b` since the chart continues to use Traefik 1.1.2.